src/test/java/hudson/plugins/parameterizedtrigger/test/BuildTriggerConfigTest.java

```diff
@@ -121,6 +126,64 @@ public void testGetProjectListStatic() throws Exception {
 
     }
 
+    public void testGetProjectListWithWorkflow() throws Exception {
+        Project<?, ?> masterProject = createFreeStyleProject("project");
+        WorkflowJob p = jenkins.createProject(WorkflowJob.class, "workflowproject");
+        p.setDefinition(new CpsFlowDefinition("println('hello')"));
```

Comment:
Actually Workflow hacks things a bit so that `println` is just an alias for `echo`. It is preferable to use `echo` directly.